libnext/src/parsers/nodes/request-parser.spec.ts

```diff
@@ -37,6 +37,12 @@ describe("@request parser", () => {
       location: expect.stringMatching(/main\.ts$/),
       line: 4
     });
+    // @ts-ignore undefined check
```

Comment:
You can use the `!` operator instead, e.g. with `expect(result.value.headers!.value)`